dist/bin/upgrade-grist

```diff
@@ -2,10 +2,19 @@
 
 set -e
 
+ENV_DIR=$(realpath $(dirname $0)/..)
+PERSIST_DIR=${PERSIST_DIR:-${ENV_DIR}/persist}
+
 docker compose pull
 
 if systemctl is-active --quiet grist.service; then
-  sudo systemctl restart grist.service
+  sudo systemctl stop grist.service
+
+  # Opened snapshots don't get cleaned up automatically
+  # TODO: Grist doc workers should do this periodically
+  find $PERSIST_DIR/grist/docs -regextype posix-extended -regex '^.+~v=.+\.grist$' -delete
```

Comment:
If this fails, the container will remain stopped. What do you think of keeping `restart` (after the cleanup) rather than doing `start + cleanup + stop`?